src_assets/windows/assets/shaders/directx/include/convert_yuv420_packed_uv_ps_base.hlsl

```diff
@@ -17,6 +17,14 @@ float2 main_ps(vertex_t input) : SV_Target
     float3 rgb_left = image.Sample(def_sampler, input.tex_right_left_center.xz).rgb;
     float3 rgb_right = image.Sample(def_sampler, input.tex_right_left_center.yz).rgb;
     float3 rgb = CONVERT_FUNCTION((rgb_left + rgb_right) * 0.5);
+#elif defined(LEFT_SUBSAMPLING_SCALE)
+    float3 rgb = image.Sample(def_sampler, input.tex_right_center_left_top.xw).rgb; // top-right
+    rgb += image.Sample(def_sampler, input.tex_right_center_left_top.yw).rgb; // top-center
+    rgb += image.Sample(def_sampler, input.tex_right_center_left_top.zw).rgb; // top-left
+    rgb += image.Sample(def_sampler, input.tex_right_center_left_bottom.xw).rgb; // bottom-right
+    rgb += image.Sample(def_sampler, input.tex_right_center_left_bottom.yw).rgb; // bottom-center
+    rgb += image.Sample(def_sampler, input.tex_right_center_left_bottom.zw).rgb; // bottom-left
+    rgb = CONVERT_FUNCTION(rgb * (1./6));
```

Comment:
```suggestion
    float3 rgb = image.Sample(def_sampler, input.tex_right_center_left_top.yw).rgb; // top-center
    rgb += image.Sample(def_sampler, input.tex_right_center_left_bottom.yw).rgb; // bottom-center
    rgb *= 2;
    rgb += image.Sample(def_sampler, input.tex_right_center_left_top.xw).rgb; // top-right
    rgb += image.Sample(def_sampler, input.tex_right_center_left_top.zw).rgb; // top-left
    rgb += image.Sample(def_sampler, input.tex_right_center_left_bottom.xw).rgb; // bottom-right
    rgb += image.Sample(def_sampler, input.tex_right_center_left_bottom.zw).rgb; // bottom-left
    rgb = CONVERT_FUNCTION(rgb * (1./12));
```
Also just realized we can sightly improve the quality and be consistent with what we do in no scaling path (https://github.com/LizardByte/Sunshine/pull/1621#discussion_r1329723658) at pretty much no additional cost.